shopelectro/views/ecommerce.py

```diff
@@ -34,6 +36,37 @@ def get_context_data(self, request, **kwargs):
         }
 
 
+# @todo #789:60m Make cart routes REST style instead of RPC.
+
+# @todo #789:15m Test http cache for cart-get route.
+
+
+def cart_etag(request):
```

Comment:
why it's not the method of the Cart class?
The Cart class directly operates with it's session too

Reply:
the question we should to ask is: why is the session in the Cart class. The Cart should know anything about http, cookies, session and etc.
Right now we haven't proper place for it, so i just have created this function.
I will create a todo for this

Reply:
I will just drop this with cache logic